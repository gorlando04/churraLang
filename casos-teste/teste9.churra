amigo "L" contribui 90
amigo "M" contribui 110
item bebida preco_L 10 quantidade_L 4
item comida preco_unit 10 quantidade 2
rateio igual