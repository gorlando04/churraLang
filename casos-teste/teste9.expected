Total do churrasco: R$60.00
L deve pagar: R$30.00
M deve pagar: R$30.00